aspnetcore/tutorials/build-your-first-blazor-app.md

```diff
@@ -21,7 +21,7 @@ This tutorial shows you how to build and modify a Blazor app. You learn how to:
 > * Use event handling and data binding in components
 > * Use dependency injection (DI) and routing in a Blazor app
 
-At the end of this tutorial, you'll have a working chat app.
+At the end of this tutorial, you'll have a working todo list app.
```

Comment:
What's a todo list app? I understand a chat app.

Reply:
I meant there is a typo. The tutorial was to teach how to build a todo list app not a chat app. But that line said 'a working chat app'